smart-contracts/osmosis/contracts/cl-vault/src/vault/swap.rs

```diff
@@ -91,10 +94,10 @@ pub fn execute_swap_non_vault_funds(
         )?);
```

Comment:
If we want this entrypoint to be permissionless, here we should probably hardcode the `forced_swap_route` to None, in order to leave the Dex Router work with its beth path.

Reply:
This is already covered by the swap_max_slippage field checking the token_out_min_amount. So we don't care if someone is hijacking the route passes through esoteric pool ids.